includes/Classifai/Command/ClassifaiCommand.php

```diff
@@ -207,6 +208,162 @@ public function text( $args = [], $opts = [] ) {
 		}
 	}
 
+	/**
+	 * Batch trigger generation of text-to-speech depending on passed-in settings.
+	 *
+	 * ## Options
+	 *
+	 * [<post_ids>]
+	 * : Comma-delimited list of post IDs to generate text-to-speech for
+	 *
+	 * [--post_type=<post_type>]
+	 * : Batch process items belonging to this post type. If not used, relies on post_ids in args
+	 *
+	 * [--post_status=<post_status>]
+	 * : Batch process items that have this post status. Default publish
+
+	 * [--per_page=<int>]
+	 * : How many items should be processed at a time. Default 100
+	 *
+	 * [--dry-run=<bool>]
+	 * : Whether to run as a dry-run. Default true
+	 *
+	 * @param array $args Arguments.
+	 * @param array $opts Options.
+	 */
+	public function text_to_speech( $args = [], $opts = [] ) {
+		$defaults = [
+			'post_type'   => false,
```

Comment:
So I've set this to `false` as a default to force someone to pass this argument in if they want to query by a particular post type; otherwise this allows us to default to the comma-separated list of post IDs. If we set this to the allowed list of post types, we'd have to change the order of operations a bit (default to looking at the passed-in post IDs first)

Reply:
Make sense.